circuits/circom/verify_signature.circom

```diff
@@ -118,11 +118,13 @@ template VerifySignature(k) {
 
   signal output valid;
   
-  component M = Hasher(k);
-  M.key <== 0;
+  component M = Hasher11();
```

Comment:
We are using Hasher11 here because it just hashes a message into a snarkBigInt. Nothing non-trivial to do with signature.